pkgs/development/python-modules/withings-api/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, poetry
+, arrow
+, requests-oauth
+, requests-oauthlib
+, typing-extensions
+, pydantic
+}:
+
+buildPythonPackage rec {
+  pname = "withings-api";
+  version = "2.4.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "withings_api"; # source and whl distribution use _ instead of -
+    inherit version;
+    sha256 = "sha256-vQ6MKeD4g4QTkXx638FW53mTkx78af7NQXF00kxgM10=";
+  };
+
+  propagatedBuildInputs = [
+    poetry
+    arrow
+    requests-oauth
+    requests-oauthlib
+    typing-extensions
+    pydantic
+  ];
+
```

```suggestion

  checkInputs = [
    unittestCheckHook
  ];
```

Please configure an explicit test hook. We really want to run upstream tests to make sure our packaging isn't broken.